src/main/java/ch/njol/skript/effects/EffInsertEntityStorage.java

```diff
@@ -0,0 +1,94 @@
+package ch.njol.skript.effects;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.bukkitutil.EntityBlockStorageUtils;
+import ch.njol.skript.bukkitutil.EntityBlockStorageUtils.EntityBlockStorageType;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.block.Block;
+import org.bukkit.block.EntityBlockStorage;
+import org.bukkit.entity.Entity;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Entity Storage Insert")
+@Description({
+	"Add an entity to an entity block storage (i.e. beehive).",
+	"The added entity must match the required type of the entity block storage (i.e. a bee).",
+	"Using a specific block storage type will restrict the blocks provided to match the type."
+})
+@Examples("add last spawned bee into the beehive storage of {_beehive}")
```

Comment:
I think I'd rather use the add/remove changers.

Reply:
Well, you can't get the entities that are inside the EntityBlockStorage. You can get the number of entities inside but that's about it.
I figured I would make it an effect so it wouldn't make users think that they could get a list of all entities inside. 
But if it's ok to make it an expression and have the get always return null, I can do that.
Let me know

Reply:
Also you cant remove a singular or specific entity, only thing you can do is clear the entities stored inside.

Reply:
Just make the expression return a number and only support clear

Reply:
So combine EffInsert, EffClear and ExprEntityCount into 1 expression?